doc/man3/flux_shell_rpc_pack.adoc

```diff
@@ -0,0 +1,48 @@
+flux_shell_rpc_pack(3)
+======================
+:doctype: manpage
+
+NAME
+----
+flux_sell_rpc_pack - Use Jansson syntax to make a remote procedure call
+
+SYNOPSIS
+--------
+
+ #include <flux/shell.h>
+ #include <errno.h>
+
+ flux_future_t *flux_shell_rpc_pack (flux_shell_t *shell, 
+                                     const char *method, 
+                                     int shell_rank, 
+                                     int flags, 
+                                     const char *fmt, 
+                                     ...);
+
+DESCRIPTION
+-----------
+
+Sends a remote procedure call `method` to rank `shell_rank`.
```

Comment:
Subtlety, this call sends an RPC to a remote *shell* by shell rank.

Reply:
Agree with outdated tag.